student.rb

```diff
@@ -6,6 +6,12 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     @classroom = classroom
   end
 
+  def classroom=(classroom)
+    @classroom = classroom
+
+    @classroom.students.push(self)
```

Comment:
- [ ] Add the `unless` in  your code, this will not allow to add the same student twice
 `classroom.students.push(self) unless classroom.students.include?(self)`